operations/mimir-mixin/config.libsonnet

```diff
@@ -119,6 +127,13 @@
     },
 
     container_names: {
+      // Microservices deployment mode. The following matchers MUST match only
+      // the instance when deployed in microservices mode (e.g. "distributor"
+      // matcher shouldn't match "mimir-write" too).
+      gateway: componentNameRegexp.gateway,
+      distributor: componentNameRegexp.distributor,
+      ingester: componentNameRegexp.ingester,
+
       // The following are container matchers used to select all components in a given "path".
       local componentsGroupMatcher = function(components) std.join('|', std.map(function(name) componentNameRegexp[name], components)),
 
```

Comment:
nit: maybe also a comment here to say that the read-write regexes may or may not match both microservices or read-write containers.

Reply:
Good idea. Done here: https://github.com/grafana/mimir/pull/3497/commits/73a07d7c12e1ca09f03b16255ebe2c5f2b75c5d6